aci/convert/convert.go

```diff
@@ -181,6 +209,16 @@ func (s serviceConfigAciHelper) getAciContainer(volumesCache map[string]bool) (c
 			EnvironmentVariables: getEnvVariables(s.Environment),
 			Resources:            resource,
 			VolumeMounts:         volumes,
+			LivenessProbe: &containerinstance.ContainerProbe{
+				Exec: &containerinstance.ContainerExec{
+					Command: healthTest,
+				},
+				InitialDelaySeconds: healthStartPeriod,
+				PeriodSeconds:       healthInterval,
+				FailureThreshold:    healthRetries,
+				SuccessThreshold:    healthRetries,
+				TimeoutSeconds:      healthTimeout,
+			},
```

Comment:
This and the above variables should be replaced by `getLivenessProbe()`, or am I missing something ?

Reply:
Wow good catch! I totally missed this one! Thanx!